Clients/GUI/Pages/UserInfo.razor

```diff
@@ -0,0 +1,133 @@
+            <InputText type="@(IsPasswordBeingChanged ? "password" : "hidden")" class="input-group-text" disabled="@(!IsPasswordBeingChanged)" @bind-Value="@user.OldPassword" />
+        </div>
+        <div class="form-group" style="@(IsPasswordBeingChanged ? "" : "height: 0; color: transparent")">
+            <label for="NewPassword">Enter new password: </label>
+            <InputText type="@(IsPasswordBeingChanged ? "password" : "hidden")" class="input-group-text" disabled="@(!IsPasswordBeingChanged)" @bind-Value="@user.NewPassword" />
+        </div>
+        <br />
+        <input type="@(IsEditModeDisabled ? "hidden" : "button")" @onclick="@ChangePasswordClickHandler" class="form-control col-3 button btn-primary" value="@(IsPasswordBeingChanged ? "Cancel changing password" : "Change password")" />
+        <br />
+        <ValidationSummary />
+        <h4 style="display:@(ErrorOccured ? "block" : "hidden"); color:red">@ErrorMessage</h4>
+        <input type="button" @onclick="@EditInfoClickHandler" class="form-control col-3 button btn-primary" value="@(IsEditModeDisabled ? "Edit" : "Save changes")" />
+    </EditForm>
+</div>
+
+@code {
+    public bool IsEditModeDisabled { get; set; }
+    public bool IsPasswordBeingChanged { get; set; }
+    public bool PopupVisible { get; set; }
+    public string ErrorMessage { get; set; }
+    public bool ErrorOccured { get; set; }
+    private UserInfoViewModel user;
+    private Guid currentId;
+
+    private void EditInfoClickHandler()
+    {
+        if (IsEditModeDisabled)
+        {
+            IsEditModeDisabled = false;
+        }
+        else // Trying to save changes
+        {
+            // Edit is inprocess in other team
+            try
+            {
+                var userInfo = new DTO.UserInformation(user.Name, user.Surname, user.DateOfBirth);
+
+                // This is in process (user/edit endpoint is another team's task)
+                if (IsPasswordBeingChanged)
+                {
+                    //UserEditor.EditUser(currentId, userInfo, user.OldPassword, user.NewPassword);
+                }
+                else
+                {
+                    //UserEditor.EditUser(currentId, userInfo);
+                }
+            }
+            // This must be changed to a custom exception later
+            catch (Exception)
+            {
+                // This will be changed to real error message
+                ErrorMessage = "error";
+                ErrorOccured = true;
+            }
+
+            Modal.Show<Modals.ChangesSavedModal>("Changes saved");
+            IsEditModeDisabled = true;
+            IsPasswordBeingChanged = false;
+        }
+    }
+
+    private void ChangePasswordClickHandler()
+    {
+        IsPasswordBeingChanged ^= true;
+    }
+
+    private void UpdateUserModel()
+    {
+        user = new UserInfoViewModel(); // TODO: get it by id (another team task)
+        user.Name = null; // to call pop-up
```

Comment:
CodeConvention: comments must be placed on a separate line

Reply:
By placing them on the same line I meant that it's temporary stuff, but agree, will change it =)